lib/widgets/receivable_btc_box.dart

```diff
@@ -30,10 +30,10 @@ class ReceivableBTCBoxState extends State<ReceivableBTCBox> {
   late final texts = context.texts();
   late final currencyState = context.read<CurrencyBloc>().state;
   late final accountState = context.read<AccountBloc>().state;
-  late final lspState = context.read<LSPBloc>().state;
 
   @override
   Widget build(BuildContext context) {
+    final lspState = context.watch<LSPBloc>().state;
```

Comment:
NIT Suggestion:

Think there's nothing stopping us from defining these in `build`:

```
    final texts = context.texts();
    final currencyState = context.read<CurrencyBloc>().state;
    final accountState = context.read<AccountBloc>().state;
    final lspState = context.read<LSPBloc>().state;
```

Please check logs & verify the UI has no issues after this change.

Reply:
Added